nexus-sdk-js/src/Organization/utils.ts

```diff
@@ -22,16 +27,34 @@ export async function createOrganization(
 }
 
 // TODO: refactor -> blocked by https://github.com/BlueBrain/nexus/issues/112
-export async function getOrganization(name: string): Promise<Organization> {
+/**
+ *
+ * @param orgLabel The organization label to fetch
+ * @param Object<revision, tag> The specific tag OR revision to fetch
+ */
+export async function getOrganization(
+  orgLabel: string,
+  options?: { revision?: number; tag?: string },
```

Comment:
These options (revision, tag) are valid for many entities (orgs, projects, resources). It makes sense to make dedicated type(s) with them, and force having one or the other since we cannot have both options set together.

To implement such a feature, I'm thinking of Conditional Types ( https://www.typescriptlang.org/docs/handbook/advanced-types.html#conditional-types , hands-on guide at http://realfiction.net/2018/06/10/typescript-type-shenanigans-conditional-types ) or Function Overloading ( https://www.typescriptlang.org/docs/handbook/functions.html#overloads )